packages/odyssey-react/src/components/Box/_box-grid.scss

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.gridTemplateColumns2 {
```

Comment:
Any ideas on how to provide these classes, but guide designers and devs to utilize bespoke grids for their features?

I know that standard grids are common and popular, but research argues that layouts should be content-specific for the best UX results.